src/build.rs

```diff
@@ -333,7 +336,7 @@ impl<'a> FingerPrint<'a> {
         let mut f = std::fs::File::open(&self.path())?;
         let mut cache_str = String::new();
         f.read_to_string(&mut cache_str)?;
-        let cache = toml::de::from_str(&cache_str)?;
+        let cache = toml::de::from_str(&cache_str).unwrap();
```

Comment:
This is potentially spurious.

Reply:
My bad, it was for debugging and I accidentally recorded it. Removed.